dask/dataframe/core.py

```diff
@@ -4223,23 +4230,40 @@ def sort_values(
         na_position: {'last', 'first'}, optional
             Puts NaNs at the beginning if 'first', puts NaN at the end if 'last'.
             Defaults to 'last'.
+        sort_function: function, optional
+            Sorting function to use when sorting underlying partitions.
+            If None, defaults to `M.sort_values` (the partition library's
+            implementation of `sort_values`).
+        sort_function_kwargs: dict, optional
+            Additional keyword arguments to pass to the partition sorting function.
+            By default, `by`, `ascending`, and `na_position` are provided.
```

Comment:
nit, but for rst we need double backticks on all these. Otherwise it's just italicshttps://dask--8345.org.readthedocs.build/en/8345/generated/dask.dataframe.DataFrame.sort_values.html

Reply:
Thanks! Always forgetful of RST code formatting 😅